advanced/pihole-FTL.service

```diff
@@ -26,6 +26,10 @@ start() {
     echo "pihole-FTL is already running"
   else
     touch /var/log/pihole-FTL.log /run/pihole-FTL.pid /run/pihole-FTL.port /var/log/pihole.log
```

Comment:
Would it not make more sense to attempt to `touch` the files _after_ running the two `mkdir`'s below?

Reply:
*cough*

Nothing to see here